Live: adding docs (#34642)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v8-0.md

Data sources can now send real-time updates to dashboards over a websocket connection. This can be used with the [MQTT data source](https://github.com/grafana/mqtt-datasource).

In addition to data source integration, events can be sent to dashboards by posting metrics to the new live endpoint: `/api/live/push` endpoint.

These metrics will be broadcast to all dashboards connected to that stream endpoint.

> See more about real-time streaming in [Grafana Live documentation]({{< relref "../live/_index.md" >}}).

Reviewer: ```suggestion
For more information about real-time streaming, refer to [Grafana Live documentation]({{< relref "../live/_index.md" >}}).
```

> See more about real-time streaming in [Grafana Live documentation chapter]({{< relref "../live/_index.md" >}}).

Reviewer 2: ```suggestion
See more about real-time streaming in [Grafana Live documentation]({{< relref "../live/_index.md" >}}).
```
